SauceExamples/Visual/README.md

```diff
@@ -0,0 +1,41 @@
+# visual-testing-worker-tests
+
+## Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting_started)
+- [Usage](#usage)
+
+## About <a name = "about"></a>
+
+These are the webdriver tests using C#.
+
+## Getting Started <a name = "getting_started"></a>
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+#### Using VS Studio
+
+* Create a new NUnit test project
+
+<img src="assets/nunit.png"/>
+
+* Add all the necessary environment variables
+  ```
+  setx SELENIUM_PROTOCOL "http"
```

Comment:
As mentioned in another comment, let's simplify and remove the `SELENIUM_PROTOCOL`, `SELENIUM_HOST`, and `SELENIUM_PORT` environment variables